frontend/src/scenes/trends/personsModalLogic.ts

```diff
@@ -146,8 +148,20 @@ export const personsModalLogic = kea<personsModalLogicType<PersonModalParams>>({
                 const filterParams = parsePeopleParams({ label, action, date_from, date_to, breakdown_value }, filters)
                 actions.setPeople(tempPeople)
                 people = await api.get(`api/person/stickiness/?${filterParams}${searchTermParam}`)
-            } else if (funnelStep) {
-                const params = { ...funnelLogic().values.filters, funnel_step: funnelStep }
+            } else if (funnelStep || filters.display === ACTIONS_LINE_GRAPH_LINEAR) {
+                let params
+                if (filters.display === ACTIONS_LINE_GRAPH_LINEAR) {
+                    // funnel trends
+                    const convertedDate = dayjs(date_from)
+                    const entrance_period_start =
+                        convertedDate.get('year') === 2001
```

Comment:
So the API currently doesn't return the year for the dates on the dataset but if we don't pass in a year, dayjs will default to `2001`, so we can probably safely assume (for now?) that a date with year `2001` isn't the actual date, and thus we should resolve it to the current year instead. And we need to pass in a year to get the funnel trends persons back for `entrance_period_start`

Reply:
updated~